src/openstack_workload_generator/entities/helpers.py

```diff
@@ -58,23 +60,28 @@ def get(key: str, regex: str = ".+", multi_line: bool = False) -> str:
 
     @staticmethod
     def load_config(config_file: str):
-        potential_profile_file = \
-            str(os.path.realpath(
-                os.path.dirname(os.path.realpath(__file__))) + f"/../../../profiles/{config_file}")
+        potential_profile_file = str(
+            os.path.realpath(os.path.dirname(os.path.realpath(__file__)))
+            + f"/../../../profiles/{config_file}"
+        )
 
         if os.path.exists(config_file):
             Config._file = config_file
-        elif not str(config_file).startswith("/") and os.path.exists(potential_profile_file):
+        elif not str(config_file).startswith("/") and os.path.exists(
+            potential_profile_file
+        ):
             Config._file = potential_profile_file
         else:
-            LOGGER.error(f"Cannot find a profile at {config_file} or {potential_profile_file}")
+            LOGGER.error(
+                f"Cannot find a profile at {config_file} or {potential_profile_file}"
+            )
             sys.exit(1)
 
         Config._file = os.path.realpath(Config._file)
 
         try:
             LOGGER.info(f"Reading {Config._file}")
-            with open(str(Config._file), 'r') as file_fd:
+            with open(str(Config._file), "r") as file_fd:
```

Comment:
This change is a noop

Reply:
Yes, that would be nice. I prefer reviews but if there is nobody how can do them, its better to have the possibility to skip them.